Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/release-lifecycle-policy.md

All releases of AC are considered stable. The LTS release channel is a subset of the stable release channel that promises additional stability, reliability, and support from our team.

> For customers looking to access Astronomer's newest features on an incremental basis, we recommend following the stable release channel and upgrading to new versions as soon as they are made generally available. Stable releases are issued roughly once per quarter for Astronomer Software and the Astro CLI.

Reviewer: ```suggestion
For customers looking to access Astronomer's newest features on an incremental basis, Astronomer recommends following the stable release channel and upgrading to new versions as soon as they are made available. Stable releases are issued approximately once per quarter for Astronomer Software and the Astro CLI.
```

For customers looking for less frequent upgrades and functional changes, we recommend following the LTS release channel. Release channels are not binding, so you are free to upgrade to any available version of Astronomer Software at any time.

> > **Note:** Release channels apply to Astronomer Software and Astronomer Certified versions. We do not currently support a long-term release channel for the Astro CLI.

Reviewer: ```suggestion
> **Note:** Release channels apply to Astronomer Software and Astronomer Certified versions. Astronomer does not currently support a long-term release channel for the Astro CLI.
```
